tests/lldp/test_lldp_syncd.py

```diff
@@ -255,7 +255,7 @@ def test_lldp_entry_table_after_lldp_restart(
         60, 2, 5, verify_lldp_table, duthost
     )  # Adjust based on LLDP service restart time
     pytest_assert(result, "no output for show lldp table after restarting lldp")
-    result = duthost.shell("sudo systemctl status lldp")
+    result = duthost.shell("sudo systemctl status lldp")["stdout"]
```

Comment:
Is the change here a duplicate of line260?

Reply:
Good catch.